spec/features/admin/budget_investments_spec.rb

```diff
@@ -284,6 +284,23 @@
       expect(page).to have_select("tag_name", options: ["All tags", "Hospitals", "Teachers"])
     end
 
+    scenario "Disable 'Calculate winner' button if incorrect phase" do
+      @budget.update(phase: 'reviewing_ballots')
+
+      visit admin_budget_budget_investments_path(@budget)
+      click_link 'Winners'
+
+      expect(page).to have_link I18n.t("admin.budgets.winners.calculate")
```

Comment:
In specs we try to use the literal english strings instead of translations, it makes the specs more readable 👌